packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -120,7 +120,13 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
       isSomeGroupExpanded: props.groupProps && !props.groupProps.isAllGroupsCollapsed
     };
 
-    this._selection = props.selection || new Selection({ onSelectionChanged: undefined, getKey: props.getKey });
+    this._selection =
+      props.selection ||
+      new Selection({
+        onSelectionChanged: undefined,
+        getKey: props.getKey,
+        selectionMode: this.props.selectionMode
```

Comment:
Super nit, but for consistency:

```suggestion
        selectionMode: props.selectionMode
```

Saves a few bytes.